### DISCUSSION TITLE
Multiple Sources in one Destination

### DISCUSSION THREAD

**Discussion by ghilmanrandhawa** [airbytehq/airbyte#34693] [Connector Questions]
Is it possible to connect multiple sources using the same connectors or different connectors with the same destination (PostgreSQL)? Currently, the schemas (table names) are uniform for different sources, and I am also struggling to find a way to segregate each source's data within the destination PostgreSQL table.

**Comment by ryjabe** [comment]
@ghilmanrandhawa this is no problem.  I would suggest doing one of the following things:

1) At the connection level configure each different source to send to a different namespace/schema.  
2) If you want them all in one schema/namespace, then you can add a prefix to the table names so that you can distinguish the source. This is also done at the Connection level.

[a comment by an automated account is not shown]
